apps/marketplace/components/BuyerRFX/BuyerRFXAdditionalInformationStage.js

```diff
@@ -61,7 +61,7 @@ class BuyerRFXAdditionalInformationStage extends Component {
         <Textfield
           model={`${this.props.model}.contactNumber`}
           label="Contact number for Marketplace support"
-          description="This number will not be visible on the Digital Marketplace. It will only be used by the Marketplace operations team in case they need to contact you."
+          description="This number will not be visible on the Digital Marketplace. It will only be used by the Marketplace operations team in case they need to contact you. Please include the area code for landline numbers."
```

Comment:
```suggestion
          description="This number will not be visible on the Digital Marketplace. It will only be used by the Marketplace operations team in case they need to contact you. Please include the area code."
```

Reply:
This is what was said to be done for the ticket and checked with our designer